test/detailed-results-MD041-MD050.md.fixed

```diff
@@ -14,7 +14,7 @@ that capitalizes the
 name wrong twice:
 markdownlint.
 
-A [normal](link) and an [empty one]() and a [fragment](#one).
+A [normal](link) and an [empty one]().
```

Comment:
Please add new `detailed-results-MD051-MD060.results.*` and a scenario for the new rule.